package.json

```diff
@@ -28,6 +28,7 @@
     "@snapshot-labs/snapshot-sentry": "^1.5.5",
     "@snapshot-labs/snapshot.js": "^0.12.28",
     "@unstoppabledomains/resolution": "^9.2.2",
+    "@web3-name-sdk/core": "^0.2.0",
```

Comment:
Isn't there a way to do onchain call to get the name instead of using that library?

Reply:
> Isn't there a way to do onchain call with Ethers.js to get the name instead of using that library?

Our library is specifically designed to achieve that functionality and also supports additional parsing features. For more details, please refer to https://docs.space.id/developer-guide/web3-name-sdk